### PR TITLE
[Examples] Test all examples on CPU

### PR DESCRIPTION
**Pull request by patrickvonplaten**
This PR accelerates the tests for:
- textual inversion 
- unconditional 
and adds new tests for
- dreambooth
- text_to_image

and makes sure those tests are also run on PRs on CPU. In my local test they should take around ~1min which is ok IMO.

### CODE REVIEW

File: .github/workflows/pr_tests.yml [hunk 1]
```diff
@@ -153,3 +165,5 @@ jobs:
       with:
         name: pr_torch_mps_test_reports
         path: reports
+
+  run_example_tests:
```
Comment (patrickvonplaten) [inline .github/workflows/pr_tests.yml:169]:
```suggestion
```

File: .github/workflows/pr_tests.yml [hunk 2]
```diff
@@ -153,3 +165,5 @@ jobs:
       with:
         name: pr_torch_mps_test_reports
         path: reports
+
```
Comment (patrickvonplaten) [inline .github/workflows/pr_tests.yml:168]:
```suggestion
```

File: examples/unconditional_image_generation/train_unconditional.py [hunk 3]
```diff
@@ -67,6 +67,12 @@ def parse_args():
         default=None,
         help="The config of the Dataset, leave as None if there's only one config.",
     )
+    parser.add_argument(
+        "--model_config_name_or_path",
+        type=str,
+        default=None,
+        help="The config of the UNet model to train, leave as None to use standard DDPM configuration.",
+    )
```
Comment (pcuenca) [inline examples/unconditional_image_generation/train_unconditional.py:75]: Nice!

**Review by pcuenca — APPROVED** [review]
Great work, thanks a lot!

**Pull request merged**